### ISSUE TITLE
question

### ISSUE THREAD

**Issue by 1zsw123** [hailanyi/VirConv#1]
Is VirConv-S.yaml right?5 epochs? I train it. But 5 performance is not good. I hope you can answer, thank you!

**Comment by hailanyi** [comment]
What's your performance, can you provide more details?
Seems that you did not use the pre-trained VirConv-T to initialize the VirConv-S. 
Please check the readme to train the VirConv-S correctly.

**Comment by 1zsw123** [comment]
ok， I really forget to train T first， sorry.

**Comment by 1zsw123** [comment]
Do I need to repeat T every time so that I can train S? I use 7 3090s to train T once, but the first time S is 90.3AP, and the second time is 88.8.

**Comment by hailanyi** [comment]
The performance is much depended on your initial VirConv-T model.
Please do not choose the best VirConv-T to initialize the VirConv-S, as the VirConv-T may be overfitted to labeled data.
Choosing a VirConv-T checkpoint with high learning rate is better.  
For example, if you trained the VirConv-T 60 epochs, you could choose the checkpoint from 40 epoch for initializing.

Besides, the performance also has some gap between different devices (based on my past experiment). But at least, I can obtain 90.5+AP.

Alternatively, the training epoch can be further tunned (such as 10 epoch, sometimes give me better results).

**Comment by 1zsw123** [comment]
OK， thank you！I will try more epochs.
